src/tpm_comm_linux.c

```diff
@@ -217,7 +217,7 @@ static void* load_abrmd(void** dylib)
         goto err;
     }
     if (size < sizeof(TCTI_CTX)) {
-        LogError("TCTI context size reported by tcti_init() in %s is too small: %lu < %lu", abrmd_name, size, sizeof(TCTI_CTX));
+        LogError("TCTI context size reported by tcti_init() in %s is too small: %zu < %zu", abrmd_name, size, sizeof(TCTI_CTX));
```

Comment:
>%zu < %zu" [](start = 80, length = 10)

Can't use zu, because the MxChip doesn't support it.  Can you do a cast to (int) and use %d?

Reply:
Cast to unsigned int and use %lu?